test/ng/qSpec.js

```diff
@@ -1586,6 +1598,8 @@ describe('q', function() {
       var rejectedPromise = q.reject('rejected');
       expect(rejectedPromise['finally']).not.toBeUndefined();
       expect(rejectedPromise['catch']).not.toBeUndefined();
+      rejectedPromise.then(noop, noop);
```

Comment:
I like `.catch(noop)` better (here and below) as it's more concoise and communicates the purpose a little better.